Tasks/DownloadGitHubReleaseV0/package.json

```diff
@@ -14,9 +14,9 @@
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
     "artifact-engine": "0.1.26",
-    "vso-node-api": "^6.2.8-preview",
-    "vsts-task-lib": "2.2.1",
-    "@types/node": "^6.0.101",
-    "typed-rest-client": "^1.0.11"
+    "vso-node-api": "6.3.1-preview",
+    "azure-pipelines-task-lib": "2.8.0",
```

Comment:
does usages like https://github.com/microsoft/azure-pipelines-tasks/blob/582d7229799ba5f766b583213539383cd4940557/Tasks/DownloadGitHubReleaseV0/main.ts#L3 need to change?